editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -535,10 +538,15 @@ const NewCanvasControlsInner = (props: NewCanvasControlsInnerProps) => {
       >
         {when(
           isSelectMode(editorMode) || isCommentMode(editorMode),
-          <SceneLabelControl
-            maybeHighlightOnHover={maybeHighlightOnHover}
-            maybeClearHighlightsOnHoverEnd={maybeClearHighlightsOnHoverEnd}
-          />,
+          <>
+            <SceneLabelControl
+              maybeHighlightOnHover={maybeHighlightOnHover}
+              maybeClearHighlightsOnHoverEnd={maybeClearHighlightsOnHoverEnd}
+            />
+            <DataReferenceParentOutline
+              targetViews={[...localSelectedViews, ...localHighlightedViews]}
```

Comment:
warning, this will blow the memoization (which by default uses `===` equality for each prop). if you pass individually the localSelectedViews and the localHighlightedViews then you get your memo back for free